From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Steven P. Goldsmith" <sgjava@gmail.com>
Date: Mon, 14 Dec 2020 18:13:44 +0100
Subject: [ARCHEOLOGY] XU4 add gpio-line-names (#2449)

> X-Git-Archeology: - Revision cfd9299122682821b79aef1507ce21f65c715037: https://github.com/armbian/build/commit/cfd9299122682821b79aef1507ce21f65c715037
> X-Git-Archeology:   Date: Mon, 14 Dec 2020 18:13:44 +0100
> X-Git-Archeology:   From: Steven P. Goldsmith <sgjava@gmail.com>
> X-Git-Archeology:   Subject: XU4 add gpio-line-names (#2449)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 9fb00809b13ad4ffd7f2938dfae309fa03993c31: https://github.com/armbian/build/commit/9fb00809b13ad4ffd7f2938dfae309fa03993c31
> X-Git-Archeology:   Date: Sun, 27 Dec 2020 18:08:56 +0100
> X-Git-Archeology:   From: Steven P. Goldsmith <sgjava@gmail.com>
> X-Git-Archeology:   Subject: XU4 fix gpio-line-names (#2503)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 161d58cca152b0238d98aa1092db95ab8bef0fd9: https://github.com/armbian/build/commit/161d58cca152b0238d98aa1092db95ab8bef0fd9
> X-Git-Archeology:   Date: Tue, 26 Sep 2023 23:36:49 +0200
> X-Git-Archeology:   From: Julian Sikorski <belegdol+github@gmail.com>
> X-Git-Archeology:   Subject: Switch odroidxu4-current kernel to 6.1 (#5716)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 88f14d1625477d737be03dc19e77245dd3f94da1: https://github.com/armbian/build/commit/88f14d1625477d737be03dc19e77245dd3f94da1
> X-Git-Archeology:   Date: Tue, 12 Dec 2023 08:31:52 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Cleanup deprecated Odroid XU4 kernel patches, cleanup old unused folders
> X-Git-Archeology:
> X-Git-Archeology: - Revision c6b77cba613e5f680ba298f8d9e194c50c223186: https://github.com/armbian/build/commit/c6b77cba613e5f680ba298f8d9e194c50c223186
> X-Git-Archeology:   Date: Mon, 06 May 2024 15:50:41 +0100
> X-Git-Archeology:   From: Julian Sikorski <belegdol+github@gmail.com>
> X-Git-Archeology:   Subject: Adapt the GPIO names patches to upstream commit 724ba67
> X-Git-Archeology:
> X-Git-Archeology: - Revision f17391b59a3a218393216a4d575a351baf6b7b43: https://github.com/armbian/build/commit/f17391b59a3a218393216a4d575a351baf6b7b43
> X-Git-Archeology:   Date: Sun, 02 Jun 2024 11:53:18 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor@armbian.com>
> X-Git-Archeology:   Subject: Odroid XU4 - adjust patch folder, apply config rewrite
> X-Git-Archeology:
---
 arch/arm/boot/dts/samsung/exynos5420-pinctrl.dtsi | 43 ++--------
 1 file changed, 7 insertions(+), 36 deletions(-)

diff --git a/arch/arm/boot/dts/samsung/exynos5420-pinctrl.dtsi b/arch/arm/boot/dts/samsung/exynos5420-pinctrl.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/boot/dts/samsung/exynos5420-pinctrl.dtsi
+++ b/arch/arm/boot/dts/samsung/exynos5420-pinctrl.dtsi
@@ -18,7 +18,6 @@ gpy7: gpy7-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPY7.0", "GPY7.1", "GPY7.2", "GPY7.3", "GPY7.4", "GPY7.5", "GPY7.6", "GPY7.7";
 	};
 
 	gpx0: gpx0-gpio-bank {
@@ -30,7 +29,6 @@ gpx0: gpx0-gpio-bank {
 		#interrupt-cells = <2>;
 		interrupts = <23 0>, <24 0>, <25 0>, <25 1>,
 			     <26 0>, <26 1>, <27 0>, <27 1>;
-		gpio-line-names = "GPX0.0", "GPX0.1", "GPX0.2", "GPX0.3", "GPX0.4", "GPX0.5", "GPX0.6", "GPX0.7";
 	};
 
 	gpx1: gpx1-gpio-bank {
@@ -42,7 +40,7 @@ gpx1: gpx1-gpio-bank {
 		#interrupt-cells = <2>;
 		interrupts = <28 0>, <28 1>, <29 0>, <29 1>,
 			     <30 0>, <30 1>, <31 0>, <31 1>;
-		gpio-line-names = "GPX1.0", "GPX1.1", "GPX1.2 CONN10 PIN 15", "GPX1.3 CONN10 PIN 18", "GPX1.4", "GPX1.5 CONN10 PIN 13", "GPX1.6 CONN10 PIN 17", "GPX1.7 CONN10 PIN 25";
+		gpio-line-names = "", "", "GPX1.2 CONN10 PIN 15", "GPX1.3 CONN10 PIN 18", "", "GPX1.5 CONN10 PIN 13", "GPX1.6 CONN10 PIN 17", "GPX1.7 CONN10 PIN 25";
 	};
 
 	gpx2: gpx2-gpio-bank {
@@ -51,7 +49,7 @@ gpx2: gpx2-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPX2.0 CONN10 PIN 26", "GPX2.1 CONN10 PIN 24", "GPX2.2", "GPX2.3", "GPX2.4 CONN10 PIN 20", "GPX2.5 CONN10 PIN 21", "GPX2.6 CONN10 PIN 19", "GPX2.7 CONN10 PIN 22";
+		gpio-line-names = "GPX2.0 CONN10 PIN 26", "GPX2.1 CONN10 PIN 24", "", "", "GPX2.4 CONN10 PIN 20", "GPX2.5 CONN10 PIN 21", "GPX2.6 CONN10 PIN 19", "GPX2.7 CONN10 PIN 22";
 	};
 
 	gpx3: gpx3-gpio-bank {
@@ -60,7 +58,7 @@ gpx3: gpx3-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPX3.0", "GPX3.1 CONN10 PIN 27", "GPX3.2 CONN11 PIN 5", "GPX3.3", "GPX3.4", "GPX3.5", "GPX3.6", "GPX3.7";
+		gpio-line-names = "", "GPX3.1 CONN10 PIN 27", "GPX3.2 CONN11 PIN 5";
 	};
 
 	dp_hpd: dp-hpd-pins {
@@ -85,7 +83,6 @@ gpc0: gpc0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPC0.0", "GPC0.1", "GPC0.2", "GPC0.3", "GPC0.4", "GPC0.5", "GPC0.6", "GPC0.7";
 	};
 
 	gpc1: gpc1-gpio-bank {
@@ -94,7 +91,6 @@ gpc1: gpc1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPC1.0", "GPC1.1", "GPC1.2", "GPC1.3", "GPC1.4", "GPC1.5", "GPC1.6", "GPC1.7";
 	};
 
 	gpc2: gpc2-gpio-bank {
@@ -103,7 +99,6 @@ gpc2: gpc2-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPC2.0", "GPC2.1", "GPC2.2", "GPC2.3", "GPC2.4", "GPC2.5", "GPC2.6";
 	};
 
 	gpc3: gpc3-gpio-bank {
@@ -112,7 +107,6 @@ gpc3: gpc3-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPC3.0", "GPC3.1", "GPC3.2", "GPC3.3";
 	};
 
 	gpc4: gpc4-gpio-bank {
@@ -121,7 +115,6 @@ gpc4: gpc4-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPC4.0", "GPC4.1";
 	};
 
 	gpd1: gpd1-gpio-bank {
@@ -130,49 +123,41 @@ gpd1: gpd1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPD1.0", "GPD1.1", "GPD1.2", "GPD1.3", "GPD1.4", "GPD1.5", "GPD1.6", "GPD1.7";
 	};
 
 	gpy0: gpy0-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY0.0", "GPY0.1", "GPY0.2", "GPY0.3", "GPY0.4", "GPY0.5";
 	};
 
 	gpy1: gpy1-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY1.0", "GPY1.1", "GPY1.2", "GPY1.3";
 	};
 
 	gpy2: gpy2-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY2.0", "GPY2.1", "GPY2.2", "GPY2.3", "GPY2.4", "GPY2.5";
 	};
 
 	gpy3: gpy3-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY3.0", "GPY3.1", "GPY3.2", "GPY3.3", "GPY3.4", "GPY3.5", "GPY3.6", "GPY3.7";
 	};
 
 	gpy4: gpy4-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY4.0", "GPY4.1", "GPY4.2", "GPY4.3", "GPY4.4", "GPY4.5", "GPY4.6", "GPY4.7";
 	};
 
 	gpy5: gpy5-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY5.0", "GPY5.1", "GPY5.2", "GPY5.3", "GPY5.4", "GPY5.5", "GPY5.6", "GPY5.7";
 	};
 
 	gpy6: gpy6-gpio-bank {
 		gpio-controller;
 		#gpio-cells = <2>;
-		gpio-line-names = "GPY6.0", "GPY6.1", "GPY6.2", "GPY6.3", "GPY6.4", "GPY6.5", "GPY6.6", "GPY6.7";
 	};
 
 	sd0_clk: sd0-clk-pins {
@@ -323,7 +308,6 @@ gpe0: gpe0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPE0.0", "GPE0.1", "GPE0.2", "GPE0.3", "GPE0.4", "GPE0.5", "GPE0.6", "GPE0.7";
 	};
 
 	gpe1: gpe1-gpio-bank {
@@ -332,7 +316,6 @@ gpe1: gpe1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPE1.0", "GPE1.1";
 	};
 
 	gpf0: gpf0-gpio-bank {
@@ -341,7 +324,6 @@ gpf0: gpf0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPF0.0", "GPF0.1", "GPF0.2", "GPF0.3", "GPF0.4", "GPF0.5";
 	};
 
 	gpf1: gpf1-gpio-bank {
@@ -350,7 +332,6 @@ gpf1: gpf1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPF1.0", "GPF1.1", "GPF1.2", "GPF1.3", "GPF1.4", "GPF1.5", "GPF1.6", "GPF1.7";
 	};
 
 	gpg0: gpg0-gpio-bank {
@@ -359,7 +340,6 @@ gpg0: gpg0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPG0.0", "GPG0.1", "GPG0.2", "GPG0.3", "GPG0.4", "GPG0.5", "GPG0.6", "GPG0.7";
 	};
 
 	gpg1: gpg1-gpio-bank {
@@ -368,7 +348,6 @@ gpg1: gpg1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPG1.0", "GPG1.1", "GPG1.2", "GPG1.3", "GPG1.4", "GPG1.5", "GPG1.6", "GPG1.7";
 	};
 
 	gpg2: gpg2-gpio-bank {
@@ -377,7 +356,6 @@ gpg2: gpg2-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPG2.0", "GPG2.1";
 	};
 
 	gpj4: gpj4-gpio-bank {
@@ -386,7 +364,6 @@ gpj4: gpj4-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPJ4.0", "GPJ4.1", "GPJ4.2", "GPJ4.3";
 	};
 
 	cam_gpio_a: cam-gpio-a-pins {
@@ -460,7 +437,7 @@ gpa0: gpa0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPA0.0 CONN10 PIN 6", "GPA0.1 CONN10 PIN 8", "GPA0.2 CONN10 PIN 4", "GPA0.3 CONN10 PIN 5", "GPA0.4", "GPA0.5", "GPA0.6", "GPA0.7";
+		gpio-line-names = "GPA0.0 CONN10 PIN 6", "GPA0.1 CONN10 PIN 8", "GPA0.2 CONN10 PIN 4", "GPA0.3 CONN10 PIN 5";
 	};
 
 	gpa1: gpa1-gpio-bank {
@@ -469,7 +446,6 @@ gpa1: gpa1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPA1.0", "GPA1.1", "GPA1.2", "GPA1.3", "GPA1.4", "GPA1.5";
 	};
 
 	gpa2: gpa2-gpio-bank {
@@ -478,7 +454,7 @@ gpa2: gpa2-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPA2.0", "GPA2.1", "GPA2.2 CONN11 PIN 4", "GPA2.3 CONN11 PIN 6", "GPA2.4 CONN10 PIN 10", "GPA2.5 CONN10 PIN 11", "GPA2.6 CONN10 PIN 9", "GPA2.7 CONN10 PIN 7";
+		gpio-line-names = "", "", "GPA2.2 CONN11 PIN 4", "GPA2.3 CONN11 PIN 6", "GPA2.4 CONN10 PIN 10", "GPA2.5 CONN10 PIN 11", "GPA2.6 CONN10 PIN 9", "GPA2.7 CONN10 PIN 7";
 	};
 
 	gpb0: gpb0-gpio-bank {
@@ -487,7 +463,6 @@ gpb0: gpb0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPB0.0", "GPB0.1", "GPB0.2", "GPB0.3", "GPB0.4";
 	};
 
 	gpb1: gpb1-gpio-bank {
@@ -496,7 +471,6 @@ gpb1: gpb1-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPB1.0", "GPB1.1", "GPB1.2", "GPB1.3", "GPB1.4", "GPB1.5", "GPB1.6", "GPB1.7";
 	};
 
 	gpb2: gpb2-gpio-bank {
@@ -505,7 +479,6 @@ gpb2: gpb2-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPB2.0", "GPB2.1", "GPB2.2", "GPB2.3";
 	};
 
 	gpb3: gpb3-gpio-bank {
@@ -514,7 +487,7 @@ gpb3: gpb3-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPB3.0", "GPB3.1", "GPB3.2 CONN10 PIN 16", "GPB3.3 CONN10 PIN 14", "GPB3.4", "GPB3.5", "GPB3.6", "GPB3.7";
+		gpio-line-names = "", "", "GPB3.2 CONN10 PIN 16", "GPB3.3 CONN10 PIN 14";
 	};
 
 	gpb4: gpb4-gpio-bank {
@@ -523,7 +496,6 @@ gpb4: gpb4-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPB4.0", "GPB4.1";
 	};
 
 	gph0: gph0-gpio-bank {
@@ -532,7 +504,6 @@ gph0: gph0-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPH0.0", "GPH0.1", "GPH0.2", "GPH0.3", "GPH0.4", "GPH0.5", "GPH0.6", "GPH0.7";
 	};
 
 	uart0_data: uart0-data-pins {
@@ -757,7 +728,7 @@ gpz: gpz-gpio-bank {
 
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		gpio-line-names = "GPZ.0 CONN11 PIN 7", "GPZ.1 CONN11 PIN 9", "GPZ.2 CONN11 PIN 11", "GPZ.3 CONN11 PIN 12", "GPZ.4 CONN11 PIN 10", "GPZ.5", "GPZ.6";
+		gpio-line-names = "GPZ.0 CONN11 PIN 7", "GPZ.1 CONN11 PIN 9", "GPZ.2 CONN11 PIN 11", "GPZ.3 CONN11 PIN 12", "GPZ.4 CONN11 PIN 10";
 	};
 
 	i2s0_bus: i2s0-bus-pins {
-- 
Armbian

